src/pivot-table/components/icons/Minus.tsx

```diff
@@ -18,7 +17,6 @@ const MinusIcon = ({ color, opacity, testid, onClick }: IconProps): JSX.Element
     xmlns="http://www.w3.org/2000/svg"
     onClick={onClick}
     data-testid={testid}
-    style={{ minWidth: "fit-content", minHeight: "fit-content" }}
```

Comment:
I recall seeing an issue where the icons would not always render with the same size. That is why the styling was added. I'll see if I can reproduce that in this branch.

Reply:
This does seem to re-introduce the issue seen in #246 

At least for me in Firefox

![Skärmavbild 2023-05-30 kl  07 58 47](https://github.com/qlik-oss/sn-pivot-table/assets/16608020/4606811a-ed0a-4504-9ac4-facb34b39ba8)

Reply:
I'll investigate more. It is specifically fit-content that isn't working in webkit